packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -50,39 +49,42 @@ export class EVMAAWallet<B extends EVMBlockchainIncludingTestnet = EVMBlockchain
         TChain,
         KernelSmartAccount<entryPoint, HttpTransport, TChain>
     >;
+    private entryPoint: EntryPoint;
     chain: B;
 
     constructor(
         account: KernelSmartAccount<EntryPoint, HttpTransport, TChain>,
         crossmintService: CrossmintWalletService,
         chain: B,
         publicClient: PublicClient,
-        ecdsaValidator: KernelValidator<entryPoint, "ECDSAValidator">
+        ecdsaValidator: KernelValidator<entryPoint, "ECDSAValidator">,
+        entryPoint: EntryPoint
```

Comment:
just fyi you can shortcut and remove line 52 by doing:
```suggestion
        private entryPoint: EntryPoint
```